boards/common/atxmega/Makefile.include

```diff
@@ -0,0 +1,41 @@
+INCLUDES += -I$(RIOTBOARD)/common/atxmega/include
+
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
```

Comment:
I think those 2 lines are not needed: serial.inc.mk is included anyway from Makefile.include and this contains the same definitions.